Dockerfile

```diff
@@ -0,0 +1,77 @@
+# renovate: datasource=repology depName=alpine_3_16/iptables
+ARG IPTABLES_VERSION=1.8.8-r1
+# renovate: datasource=github-releases depName=kubernetes/dns
+ARG KUBERNETES_DNS_VERSION=1.21.1
+
+
+FROM docker.io/library/alpine:3.16 as iptables-installer
+ARG IPTABLES_VERSION
+ENV IPTABLES_VERSION="${IPTABLES_VERSION:?}"
```

Comment:
Isn't that redundant to the line above?

`ARG IPTABLES_VERSION` already should provide IPTABLES_VERSION as env?

Reply:
Yes, setting the ENV is not necessary to build the image. I use it to persist the value of the ARG in the container image to make it easier to see which versions were used during build. Since this is only a build-stage and not the final image it is not really useful, unless someone decides to sets this stage as a build target. Do you want me to remove it?

Reply:
After discussion & understanding it, keeping them so they are persisted in the ENV seems like a nice think to have

Reply:
```
ARG IPTABLES_VERSION
# include build ARG also as ENV (to keep info about value in image)
# reason: ARG instruction values are only available as environment variables during build-time,
#              not anymore during run-time of a container based on said build image.
ENV IPTABLES_VERSION="${IPTABLES_VERSION:?}"
```